### PR TITLE
Added RL and Multi-Objective examples to `examples/README.md`

### PR DESCRIPTION
**Pull request by jeromepatel**
## Motivation
With reference to issue #2306, I have added the link for RL and Multi-objective examples to the <code>examples/README.md</code> file.

## Description of the changes
Previously unlinked three examples were added to the readme file of the example folder. Let me know if I need to add more modifications while working on <code>examples/README.md</code> file.
Thank you!

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@Crissman Could you review this PR if you have time?

File: examples/README.md [hunk 1]
```diff
@@ -88,6 +93,10 @@ In addition, integration modules are available for the following libraries, prov
 
 * [Optimizing on Kubernetes](./kubernetes/README.md)
 
+### Examples of Reinforcement Learning
+
+* [Optimizating Hyperparameters for Stable-Baslines Agent](./rl/sb3_simple.py)
```
Comment (HideakiImamura) [inline examples/README.md:98]: Fix typo
```suggestion
* [Optimization of hyperparameters for Stable-Baslines Agent](./rl/sb3_simple.py)
```

Comment (jeromepatel) [reply examples/README.md:98]: Thank you for pointing that out, I'll change that.

**Review by HideakiImamura — APPROVED** [review]
LGTM.

**Review by Crissman — APPROVED** [review]
LGTM!

**Pull request merged**